scalyr_agent/platform_windows.py

```diff
@@ -271,8 +243,39 @@ def __init__(self):
         # The file path to the configuration.  We need to stash this so it is available when start is invoked.
         self.__config_file_path = None
 
+        def all_admin_names():
```

Comment:
I know it would be hard to write end to end tests for this function, but at least some mocking based unit tests would be nice :)